tasks/symlink.yml

```diff
@@ -1,4 +1,4 @@
 ---
 # Performs symlink exchange
-- name: Change softlink to new release
-  file: state=link path={{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }} src={{ release_path.stdout }}
+- name: ANSISTRANO | Change softlink to new release
+  file: state=link path={{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }} src={{ ansistrano_ansistrano_release_path.stdout }}
```

Comment:
Should be ansistrano_release_path instead of ansistrano_ansistrano_release_path (same for tasks/update-code.yml)